modules/ROOT/pages/notifications/gqlstatus-info-codes.adoc

```diff
@@ -0,0 +1,51 @@
+:description: The page contains GQLSTATUS info codes producing results such as SUCCESS, NO DATA, and OMITTED RESULT.
+
+[[gql-status-info-codes]]
+= General GQLSTATUS codes
+
+GQL requires several new statuses that are not covered by the current Neo4j notification framework.
+
+.GQL status information codes
+[cols="1,1,1,1", options="header"]
+|===
+|GQLSTATUS
+|Condition
+|Subcondition
+|Description
+
+| 00000
+| Successful completion
+|
+| Successful completion with a regular non-empty result
+
+| 00001
+| Successful completion
+| Omitted result
+| Successful completion with no return columns (e.g., update with no `RETURN` clause)
+
+| 02000
+| No data
+| 
+| Successful completion with an empty result
+|===
+
+In summary:
+
+* _successful completion_ (00000) is raised to indicate successful completion with a regular non-empty result.
+* _successful completion — omitted result_ (00001) is raised to indicate successful completion with an omitted result.
+* _no data_ (02000) is raised to indicate successful completion with an empty result.
+
+In the documentation, these results are referred to as `SUCCESS`, `OMITTED RESULT`, and `NO DATA`.
+
+All query executions always either return one of `SUCCESS`, `OMITTED RESULT`, or `NO DATA` results or result in an error.
+
+link:https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/[`EXPLAIN` queries] always give `OMITTED RESULT`.
+
+Classification, severity, position, and status parameters are not meaningful for these GQL statuses, so they are not included in the diagnostic record but the corresponding getter methods on server and driver side will be poly-filled with the default values.
+
+These statuses are filled in by the server unless the server is too old to be aware of GQL-status objects (the case described in **Compatibility issues**), then it will be filled by the driver.
```

Comment:
I'd like to link this paragraph to the place where we'll talk about the compatibility between old server and new drivers.